### ISSUE TITLE
Fix Homebrew Channel /exec bug on webOS 1

### ISSUE THREAD

**Issue by throwaway96** [throwaway96/webos-homebrew-channel#3]

**Comment by throwaway96** [comment]
Done in webosbrew#172.

**Comment by throwaway96** [comment]
Actually not done. On webOS 1, you don't get the base64 output in the response.

webOS 1.4.0:
```
# luna-send -n 1 'luna://org.webosbrew.hbchannel.service/exec' '{"command":"echo test"}'
{"returnValue":true,"error":null,"stdoutString":"test\n","stdoutBytes":"test\n","stderrString":"","stderrBytes":""}
```

webOS 8.1.0:
```
# luna-send -n 1 'luna://org.webosbrew.hbchannel.service/exec' '{"command":"echo test"}'
{"returnValue":true,"error":null,"stdoutString":"test\n","stdoutBytes":"dGVzdAo=","stderrString":"","stderrBytes":""}```

**Comment by throwaway96** [comment]
Fixed in webosbrew#178.
